app/controllers/schemes_controller.rb

```diff
@@ -3,11 +3,11 @@ class SchemesController < ApplicationController
   include Modules::SearchFilter
 
   before_action :authenticate_user!
-  before_action :find_resource, except: %i[index create new changes]
+  before_action :find_resource, except: %i[index create new changes email_csv download_csv csv_confirmation]
   before_action :redirect_if_scheme_confirmed, only: %i[primary_client_group confirm_secondary_client_group secondary_client_group support details]
-  before_action :authorize_user
-  before_action :session_filters, if: :current_user, only: %i[index]
-  before_action -> { filter_manager.serialize_filters_to_session }, if: :current_user, only: %i[index]
+  before_action :authorize_user, except: %i[email_csv download_csv csv_confirmation]
```

Comment:
How come we don't want to authorise for download_csv and csv_confirmation?

Reply:
There isn't really anything to authorize - all user types can download their visible schemes so if I added a policy it would just check the user is of any type... or should the authorization be doing something further? I think we don't have an authorization for the labels version of log downloads which are available to non-support users.